test/check-application

```diff
@@ -791,6 +796,47 @@ class TestFiles(testlib.MachineCase):
         b.click(".breadcrumb-button:nth-of-type(3)")
         m.execute("chattr -i /home/admin/locked")
 
+    def testKeyboardNav(self):
+        b = self.browser
+        m = self.machine
+
+        self.enter_files()
+
+        create_files = ""
+        for i in range(0, 4):
+            create_files += f"touch /home/admin/file{i}; "
+        m.execute(create_files)
+
+        # HACK: Focus the iframe for global keybindings
```

Comment:
Is it really a hack? Is there a cleaner solution, like focussing an actual element?

Reply:
Not really a hack, but I would love our tests to do it for us but that seems like an impossible catch 22 :)